### DISCUSSION TITLE
Tag torrents that contain certain file types

### DISCUSSION THREAD

**Discussion by NateH2O** [StuffAnThings/qbit_manage#641] [Ideas]
I'd love to have any torrents that contain rar files to be tagged with "rar".

Could work similar to the tracker tagging. Supply substrings to match on, if matched add tag provided.

The actual problem I'm trying to solve is that I run the "no hardlink check" and it tags things with noHL, but many times that is because it contains a rar file that unpackarr should have already dealt with.

**Comment by cgetzen** [comment]
I have submitted a PR for this here: https://github.com/StuffAnThings/qbit_manage/pull/1037

**Comment by tnoetzel** [comment]
@NateH2O @cgetzen - Feels like a lot of this logic could be expanded to support other use cases like tagging based on torrent names. Added #1070
